Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/credentialprovider/keyring.go

```diff
@@ -111,6 +111,11 @@ func isDefaultRegistryMatch(image string) bool {
 		return true
 	}
 
+	if strings.HasSuffix(parts[0], "docker.io") {
```

Comment:
good catch, will fix that also in upstream PR, I think we have to be specific as @ncdc suggested